src/plugins/intel_gpu/src/kernel_selector/kernels/permute/permute_kernel_ref.cpp

```diff
@@ -100,6 +100,7 @@ static std::string GetReorderedOutputOrder(const permute_params& params, const s
 
 ParamsKey PermuteKernelRef::GetSupportedKey() const {
     ParamsKey k;
+    k.EnableInputDataType(Datatype::BF16);
```

Comment:
does permute support bf16 input? I don't see any change from permute cl kernel..

Reply:
it is needed as permute is done to transpose matrix before multiply
![image](https://github.com/openvinotoolkit/openvino/assets/161812866/7517cee1-eb8e-435d-bba0-9f314ab315a7)